### PR TITLE
Add episode 42

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

File: _episodes/42.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+* Support for `DELETE` in MongoDB.
+* Faster aggregations.
+* Faster data transfers with fault-tolerant execution.
+* Faster `SHOW SCHEMAS` in BigQuery.
+* Faster `expire_snapshots` in Apache Iceberg.
+
+Additional highlights worth a mention according to Cole:
+
+*
```
Comment (colebow) [inline _episodes/42.md:71]: I think we can skip this because we didn't talk about anything extra.

File: _episodes/42.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+More detailed information is available in the release notes for
+[Trino 402]({{site.url}}/docs/current/release/release-402.html),
+and
+[Trino 403]({{site.url}}/docs/current/release/release-403.html).
+
+## Trino Summit 2022 recap
+
+This episode we're doing a recap of both the Trino Summit and the first Trino
+Contributor Congregation. We dive into what everyone's favorite Trino Summit
+sessions are. Then we finally cover key takeaways from the Trino Contributor
```
Comment (colebow) [inline _episodes/42.md:82]:
```suggestion
Contributor Congregation. We dive into what everyone's favorite Trino Summit
sessions were. Then we cover key takeaways from the Trino Contributor
```

File: _episodes/42.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+Congregation that took place the day after and some of the topics we went over
```
Comment (colebow) [inline _episodes/42.md:83]:
```suggestion
Congregation, which took place the day after and some of the topics we went over
```

**Review by colebow — APPROVED** [review]

File: _episodes/42.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+[Trino 402]({{site.url}}/docs/current/release/release-402.html),
+and
+[Trino 403]({{site.url}}/docs/current/release/release-403.html).
+
+## Trino Summit 2022 recap
+
+This episode we're doing a recap of both the Trino Summit and the first Trino
+Contributor Congregation. We dive into what everyone's favorite Trino Summit
+sessions were. Then we cover key takeaways from the Trino Contributor
+Congregation, which took place the day after and some of the topics we went over
```
Comment (mosabua) [inline _episodes/42.md:81]: after.

and get rid of the rest

**Review by mosabua — CHANGES_REQUESTED** [review]
One quick update .. then ready

**Review by mosabua — APPROVED** [review]

**Pull request merged**